korman/properties/modifiers/region.py

```diff
@@ -203,6 +203,237 @@ def requires_actor(self):
+                ("kFlagReflectionsScale", "Reflections Scale", "Reflection level"),
+                ("kFlagReflectionsDelayScale", "Reflections Delay Scale", "Initial reflection delay time"),
+                ("kFlagReverbScale", "Reverb Scale", "Reverberation level"),
+                ("kFlagReverbDelayScale", "Reverb Delay Scale", "Late reverberation delay time"),
+                ("kFlagEchoTimeScale", "Echo Time Scale", "Echo time"),
+                ("kFlagModulationTimeScale", "Modulation Time Scale", "Modulation time"),
+                ("kFlagDecayHFLimit", "Decay HF Limit", "Limit unnaturally long decay times of high-frequency sounds by forcing a limit to the decay time to be calculated from the Air Absorption HF value")]
+
+class PlasmaReverbRegion(PlasmaModifierProperties):
+    pl_id = "reverb"
+    pl_depends = {"softvolume"}
+
+    bl_category = "Region"
+    bl_label = "Sound Reverb Region"
+    bl_description = "Sound Reverberation (EAX) Region"
+    bl_icon = "IPO_ELASTIC"
+
+    preset = EnumProperty(name="Environment Preset",
+                          description="The type of audio environment to simulate",
+                          items=[("GENERIC", "Generic", "A generic-sounding environment with light reverberation"),
+                                 ("PADDEDCELL", "Padded cell", ""),
+                                 ("ROOM", "Room", ""),
+                                 ("BATHROOM", "Bathroom", ""),
+                                 ("LIVINGROOM", "Living room", ""),
+                                 ("STONEROOM", "Stone room", ""),
+                                 ("AUDITORIUM", "Auditorium", ""),
+                                 ("CONCERTHALL", "Concert Hall", ""),
+                                 ("CAVE", "Cave", ""),
+                                 ("ARENA", "Arena", ""),
+                                 ("HANGAR", "Hangar", ""),
+                                 ("CARPETTEDHALLWAY", "Carpetted hallway", ""),
+                                 ("HALLWAY", "Hallway", ""),
+                                 ("STONECORRIDOR", "Stone corridor", ""),
+                                 ("ALLEY", "Alley", ""),
+                                 ("FOREST", "Forest", ""),
+                                 ("CITY", "City", ""),
+                                 ("MOUNTAINS", "Mountains", ""),
+                                 ("QUARRY", "Quarry", ""),
+                                 ("PLAIN", "Plain", ""),
+                                 ("PARKINGLOT", "Parking lot", ""),
+                                 ("SEWERPIPE", "Sewer pipe", ""),
+                                 ("UNDERWATER", "Underwater", ""),
+                                 ("DRUGGED", "Drugged", ""),
+                                 ("DIZZY", "Drizzy", ""),
+                                 ("PSYCHOTIC", "Psychotic", ""),
+                                 ("MORE", "More choices...", ""),
```

Comment:
I think it might be better to use a checkbox (read: BoolProperty) somewhere to control displaying more options in the same drop-down.

Reply:
As in, a preference in the Blender user settings (next to the game/python configuration) ? I fear most users will never find it and assume only the first part of the list is available, and it might affect sharing Blender files across installs. Maybe having both lists merged into one by default would be better ? The way Blender displays lists should make it readable enough.

I'm fine with either solution, though. Let me know what you prefer.

Reply:
Sorry about neglecting to respond. I think merging them all into one list would be fine.